packages/x-bus/src/x-bus.types.ts

```diff
@@ -0,0 +1,144 @@
+import { Dictionary, Keys } from '@empathyco/x-utils';
+import { Observable, Subject } from 'rxjs';
+
+/**
+ * Alias representing a
+ * {@link https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#return_value | timeoutID}
+ * value.
+ *
+ * @internal
+ */
+export type TimeoutId = number;
+
+/**
+ * Alias representing a positive number to represent the priority of an {@link XEvent} in an
+ * {@link XPriorityBus}.
+ *
+ * @internal
+ */
+export type Priority = number;
+
+/**
+ * Alias representing a dictionary where the key is the name of the {@link XEvent} and the type
```

Comment:
This description is not right if you are using a default dictionary, because the key can be anything not a XEvent.

Reply:
+1

Reply:
Yep, should be 'SomeEvent' or something like that

Reply:
Renamed